core/src/com/fourinachamber/fortyfive/map/detailMap/MapEvent.kt

```diff
@@ -18,13 +16,18 @@ object MapEventFactory {
         "EncounterMapEvent" to { EncounterMapEvent(it) },
         "EnterMapMapEvent" to { EnterMapMapEvent(it.get<String>("targetMap"), it.get<Boolean>("placeAtEnd")) },
         "NPCMapEvent" to { NPCMapEvent(it.get<String>("npc")) },
-        "ShopMapEvent" to { it ->
+        "ShopMapEvent" to {
             ShopMapEvent(
                 it.get<String>("type"),
-                it.get<String>("biome"),
                 it.get<String>("person"),
                 it.get<Long?>("seed") ?: (Math.random() * 1000).toLong(),
-                it.get<List<OnjInt>>("boughtIndices").map{it2->it2.value.toInt()}.toMutableSet(),
+                it.get<List<OnjInt>>("boughtIndices").map { it2 -> it2.value.toInt() }.toMutableSet(),
```

Comment:
when nesting lambdas like this, please give the parameters proper names, not `it2`